dynamiqs/utils/quantum_utils/general.py

```diff
@@ -264,18 +266,23 @@ def ptrace(x: ArrayLike, keep: int | tuple[int, ...], dims: tuple[int, ...]) ->
 
     Examples:
         >>> psi_abc = dq.tensor(dq.fock(3, 0), dq.fock(4, 2), dq.fock(5, 1))
+        >>> psi_abc.dims
+        (3, 4, 5)
         >>> psi_abc.shape
         (60, 1)
-
-        # todo: temporary fix
-        # >>> rho_a = dq.ptrace(psi_abc, 0, (3, 4, 5))
-        # >>> rho_a.shape
-        # (3, 3)
-        # >>> rho_bc = dq.ptrace(psi_abc, (1, 2), (3, 4, 5))
-        # >>> rho_bc.shape
-        # (20, 20)
+        >>> rho_a = dq.ptrace(psi_abc, 0, (3, 4, 5))
+        >>> rho_a.dims
+        (3,)
+        >>> rho_a.shape
+        (3, 3)
+        >>> rho_bc = dq.ptrace(psi_abc, (1, 2), (3, 4, 5))
```

Comment:
```suggestion
        >>> rho_bc = dq.ptrace(psi_abc, (1, 2))
```

Reply:
Might be useful to add an example with the use of explicit `dims` afterwards, but the basic example should be without.